make/photon/db/upgrade.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
```

Comment:
if you use `set -e`

There may not be a chance to see the log messages in `/tmp` directory?

Reply:
yes, this should be removed.